src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -26,6 +29,22 @@ export default function usePaymentInformation (_, context) {
     return payments && payments[0].paymentMethod === PaymentMethods.CHEQUE
   })
 
+  function adjustRoutingSlipChequeNumber (num: string, i: number = 0) {
+    const chequeNumToChange = {
+      chequeNum: num,
+      idx: i
```

Comment:
what's the difference between id and idx?

Reply:
it's the index, just changed to paymentIndex